package.json

```diff
@@ -10,6 +10,10 @@
   "description": "Angular.js wrapper for Dan Grosmann's bootstrap date range picker (https://github.com/dangrossman/bootstrap-daterangepicker).",
   "license": "MIT",
   "private": false,
+  "dependencies": {
+    "bootstrap-daterangepicker": "1.3.23"
```

Comment:
I know this has been already merged now, but wondering if this dependency should point to `2.1.24` ?